imnodes.h

```diff
@@ -171,6 +173,13 @@ void Shutdown();
 // function sets the GImGui global variable, which is not shared across dll boundaries.
 void SetImGuiContext(ImGuiContext* ctx);
 
+Context* ContextCreate();
+void ContextFree(Context*);
```

Comment:
These functions instead of `Initialize()` and `Shutdown()`. Also slightly adjsuted the naming to match `ImGui`.

```suggestion
Context* CreateContext();
void DestroyContext(Context* = NULL);
```